core/src/main/scripts/importer/validateData.py

```diff
@@ -2437,21 +2437,30 @@ def processCaseListDirectory(caseListDir, cancerStudyId, logger,
                     'White space in sample id is not supported',
                     extra={'filename_': case,
                            'cause': value})
-                
 
-    for required_id in required_stable_ids:
-        if required_id not in stableid_files:
-            if required_id == cancerStudyId + '_all':
-                suggestion = ("Consider adding 'add_global_case_list: true' "
-                              "to the study metadata file")
-            else:
-                suggestion = "Please define it in the 'case_lists' folder"
-            logger.error("No  case list found for stable_id '%s'. %s",
-                         required_id,
-                         suggestion)
+    logger.info('Validation of case list folder complete')
+
+    return stableid_files
+
 
-    logger.info('Validation of case lists complete')
+def validate_defined_caselists(cancer_study_id, case_list_fns, file_types, logger):
 
+    """Validate the set of case lists defined in a study.
+
+    Args:
+        cancer_study_id (str): the study ID to be expected in the stable IDs
+        case_list_fns (Iterable[str]): stable ids of defined case lists
+        file_types (Dict[str, str]): listing of the MetaFileTypes with high-
+            dimensional data in this study--these may imply certain case lists
+        logger: logging.Logger instance to log output to
+    """
+
+    if cancer_study_id + '_all' not in case_list_fns:
```

Comment:
No, in that case this dictionary will list that the case list with suffix `_all` was defined in the study metadata file.